Monika After Story/game/script-topics.rpy

```diff
@@ -10501,4 +10501,159 @@ label monika_hemispheres_nogets_snow:
     m 3hksdlb "At least you don't have to worry about shoveling it."
     call monika_snow_nogets_snow
     return
+    
+    default persistent._mas_pm_wants_children = None
+    # do you want children?
+    
+    default persistent._mas_pm_wants_1_child = None
+    # do you want one child?
+    
+    default persistent._mas_pm_wants_boy = None
+    # do you want a boy?
+    
+    default persistent._mas_pm_wants_girl = None
+    # do you want a girl?
+    
+    default persistent._mas_pm_wants_2_to_3_children = None
+    # do you want two to three children?
+    
+    default persistent._mas_pm_wants_many_children = None
+    # do you want many children?
+    
+    default persistent._mas_pm_wants_children = None
+    # do you not want children?
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="Children",category=['romance','monika','life'],prompt="Having Children",random=True))
+
+label monika_children:
+    if mas_isMoniLOVE(higher=True):
+    m 3rkc "Hey, [player]..."
+    m 3rkbsd "Did you know when couples are close enough they start talking about having children?"
+    m 3rkbfsdlc "O-Of course you knew that, Ahaha~" 
+    m 2dkbfc "..." 
+    m 4rsbfsdla "Just a thought I had, you know..." 
+    show monika 5rsbfsdld at t11 zorder MAS_MONIKA_Z with dissolve
+    m 5rsbfsdld "I was wondering if-" 
+    m 2dsbfc "..." 
+    m 3hubfsdlb "Ahaha. I thought this would be easier to talk about with you..." 
+    m 4etbfc "I guess I'll get to the point."
+    menu:
+        m 2esbfsdlc "When I make it out there to your reality, would you like to have children with me?"
```

Comment:
As a note, when putting a line in a menu like this, you'll need to `show monika expression_here` *before* the menu, and the line inside the menu should just be `m "Question"`